cpu/cpustress.py

```diff
@@ -116,7 +116,7 @@ def __cpu_toggle(core):
     @staticmethod
     def __kill_process(pids):
         for pid in pids:
-            process.run("kill -9 %s" % pid)
+            process.run("kill -9 %s" % pid, ignore_status=True)
```

Comment:
@abdhaleegit both the approaches does not catch the failure and do anything with it. Is that intentional / fine ?

Reply:
It is fine, as we are setting cpu affinity in random way, it is expected that process gets terminated...so at the time of cleanup 'kill' command fails if pid not found.. making the test fail.

so we are not catching any failure here.